app/experimenter/experiments/constants/nimbus.py

```diff
@@ -206,6 +206,31 @@ class NimbusTargetingConfig:
     application_choice_names=(Application.DESKTOP.name,),
 )
 
+TARGETING_WINDOWS_WITH_USERCHOICE = NimbusTargetingConfig(
+    name="Users on Windows with UserChoice support (Windows build 15063 or newer)",
+    slug="windows_userchoice",
+    description="Users on Windows with UserChoice support (i.e., Windows build 15063+)",
+    targeting="os.windowsBuildNumber >= 15063",
+    desktop_telemetry="environment.system.os.windows_build_number >= 15063",
+    application_choice_names=(Application.DESKTOP.name,),
+)
+
+TARGETING_WINDOWS_WITH_USERCHOICE_FIRST_RUN = NimbusTargetingConfig(
+    name="First start-up users on Windows with UserChoice support "
+    "(Windows build 15063 or newer)",
+    slug="windows_userchoice_first_run",
+    description="First start-up users (e.g. for about:welcome) on Windows with "
+    "UserChoice support (i.e., Windows build 15063+)",
+    targeting="{first_run} && {user_choice}".format(
```

Comment:
This expands to `((isFirstStartup && !('trailhead.firstrun.didSeeAboutWelcome'|preferenceValue)) || experiment.slug in activeExperiments) && os.windowsBuildNumber >= 15063` so I think it should be fine as users probably aren't often downgrading from 10+ to 7/8 (and getting unenrolled).